test/index.js

```diff
@@ -0,0 +1,17 @@
+var test = require('tape'),
```

Comment:
these tests aren't particularly readable for a student to look at and understand how this works. They are familiar with Jasmine - will that testing framework work here? or is there a way to refactor or add comments to simplify?